client/payment-details/dispute-details/dispute-summary-row.tsx

```diff
@@ -105,34 +97,7 @@ const DisputeSummaryRow: React.FC< Props > = ( { dispute, daysRemaining } ) => {
 		{
 			title: __( 'Respond By', 'woocommerce-payments' ),
 			content: (
-				<span className="dispute-summary-row__response-date">
-					{ respondByDate }
-					<span
-						className={ classNames( {
-							'dispute-summary-row__response-date--urgent':
-								daysRemaining < 3,
-							'dispute-summary-row__response-date--warning':
-								daysRemaining < 7 && daysRemaining > 2,
-						} ) }
-					>
-						{ daysRemaining > 0 &&
-							sprintf(
-								// Translators: %s is the number of days left to respond to the dispute.
-								_n(
-									'(%s day left to respond)',
-									'(%s days left to respond)',
-									daysRemaining,
-									'woocommerce-payments'
-								),
-								daysRemaining
-							) }
-
-						{ daysRemaining === 0 &&
-							__( '(Last day today)', 'woocommerce-payments' ) }
-						{ daysRemaining < 0 &&
-							__( '(Past due)', 'woocommerce-payments' ) }
-					</span>
-				</span>
+				<DisputeDueByDate dueBy={ dispute.evidence_details.due_by } />
```

Comment:
🔥 🚀 🏅

Reply:
Oh god sorry for the late reply on this I get lost in the Github threads, please forgive me. I agree with your comment @haszari it should appear more as a stamp instead of an insert in that last bullet point.

Reply:
> Can we please remove, or do I need to create an issue?

@nikkivias if you can make an issue to get us started we'd really appreciate it! We can handle this as maintenance, should be a relatively painless fix.

> On another note, I'm curious as you talk about the extra line at the bottom, why are the buttons not visible here?

I suspect this is an old screenshot from when those specific action buttons were not implemented. Feel free to test and if you see any gaps chat in slack or just make an issue 🙌